### PR TITLE
Adds Lorem BR

### PR DESCRIPTION
**Pull request by abnersajr**
This PR adds Lorem BR into the lib
Based in the `Lorem.rb` default version
Documentation also was updated.

### CODE REVIEW

File: test/test_lorem_br.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+class TestLoremBR < Test::Unit::TestCase
+  include DeterministicHelper
+
+  assert_methods_are_deterministic(
+    FFaker::LoremBR,
+    :paragraph, :sentence, :phrase, :paragraphs, :sentences, :phrases, :words, :word, :characters
+  )
+
+  WORD = /[\p{L}\p{N}]+/i
+  WORDS = /[ \p{L}\p{N}]+/i
```
Comment (marocchino) [inline test/test_lorem_br.rb:14]: There are two problems with this.

1. This regular expression does not cover the entire string, so any string including this character is acceptable. likes:
  - negócios안녕
  - おはようbr
2. this regexp does not cover some words in your dictionary like:
  - mestre-construtor
  - disso?

I recommend not putting punctuation marks in the word list, and regular expressions to scan the entire string.

Comment (abnersajr) [reply test/test_lorem_br.rb]: Added a more strict regex and removed the word with a punctuation marks.

File: test/test_lorem_br.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+class TestLoremBR < Test::Unit::TestCase
+  include DeterministicHelper
+
+  assert_methods_are_deterministic(
+    FFaker::LoremBR,
+    :paragraph, :sentence, :phrase, :paragraphs, :sentences, :phrases, :words, :word, :characters
+  )
+
+  WORD = /[A-zÀ-ü-]+/i
+  WORDS = /[ A-zÀ-ü-]+/i
```
Comment (marocchino) [inline test/test_lorem_br.rb:14]:
```suggestion
  WORD = /\A[A-zÀ-ü-]+\z/i
  WORDS = /\A[ A-zÀ-ü-]+\z/i
```

Comment (abnersajr) [reply test/test_lorem_br.rb]: Done.

File: test/test_lorem_br.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  def test_words
+    assert_match(WORDS, FFaker::LoremBR.words.join(' '))
+  end
+
+  def test_word
+    assert_match(WORD, FFaker::LoremBR.word)
+  end
+
+  def test_characters
+    assert_match(WORD, FFaker::LoremBR.characters)
```
Comment (marocchino) [inline test/test_lorem_br.rb]: It failed because characters includes numbers.
should we just remove it or add another regexp?

Comment (abnersajr) [reply test/test_lorem_br.rb]: I just added a new regex to the characters test.

**Review by marocchino — APPROVED** [review]
💯

**Comment by marocchino** [comment]
Thanks!

**Pull request merged**